react/features/base/tracks/actions.native.ts

```diff
@@ -47,12 +45,6 @@ export function toggleScreensharing(enabled: boolean, _ignore1?: boolean, _ignor
  * @returns {void}
  */
 async function _startScreenSharing(dispatch: IStore['dispatch'], state: IReduxState) {
-    const pipWhileScreenSharingEnabled = getFeatureFlag(state, PIP_WHILE_SCREEN_SHARING_ENABLED, false);
-
-    if (!pipWhileScreenSharingEnabled) {
-        setPictureInPictureEnabled(false);
```

Comment:
There is another call to this in line 74, pl remove it. It also needs to go from `toggleScreenSharing`